### PR TITLE
Update acknowledgements for v0.9-alpha

### PR DESCRIPTION
**Pull request by ewgenius**

### CODE REVIEW

**Review by lukekim — DISMISSED** [review]

File: acknowledgements.md [hunk 1]
```diff
@@ -44,70 +44,95 @@ gopkg.in/yaml.v3, https://github.com/go-yaml/yaml/blob/v3.0.1/LICENSE, MIT
...
+| serde_yaml | 0.9.32 | David Tolnay <dtolnay@gmail.com> | https://github.com/dtolnay/serde-yaml | Apache-2.0 OR MIT |  | YAML data format for Serde |
 | snafu | 0.7.5 | Jake Goulding <jake.goulding@gmail.com> | https://github.com/shepmaster/snafu | Apache-2.0 OR MIT |  | An ergonomic error handling library |
-| snafu | 0.8.0 | Jake Goulding <jake.goulding@gmail.com> | https://github.com/shepmaster/snafu | Apache-2.0 OR MIT |  | An ergonomic error handling library |
-| spiced | 0.7.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  | Spice OSS |
-| spicepod | 0.7.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  |  |
-| sql_provider_datafusion | 0.7.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  |  |
+| snafu | 0.8.2 | Jake Goulding <jake.goulding@gmail.com> | https://github.com/shepmaster/snafu | Apache-2.0 OR MIT |  | An ergonomic error handling library |
+| spiced | 0.9.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  | Spice OSS |
```
Comment (lukekim) [inline acknowledgements.md]: We can exclude spice from the list

File: acknowledgements.md [hunk 2]
```diff
@@ -44,70 +44,95 @@ gopkg.in/yaml.v3, https://github.com/go-yaml/yaml/blob/v3.0.1/LICENSE, MIT
...
+| spicepod | 0.9.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  |  |
+| sql_provider_datafusion | 0.9.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  |  |
```
Comment (lukekim) [inline acknowledgements.md]: Also our crates

File: acknowledgements.md [hunk 3]
```diff
@@ -44,70 +44,95 @@ gopkg.in/yaml.v3, https://github.com/go-yaml/yaml/blob/v3.0.1/LICENSE, MIT
 | name | version | authors | repository | license | license_file | description |
 |------|---------|---------|------------|---------|--------------|-------------|
 | name | version | authors | repository | license | license_file | description |
-| app | 0.7.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  | All loaded components from the root Spicepod and its dependencies |
-| arrow | 49.0.0 | Apache Arrow <dev@arrow.apache.org> | https://github.com/apache/arrow-rs | Apache-2.0 |  | Rust implementation of Apache Arrow |
-| arrow-flight | 49.0.0 | Apache Arrow <dev@arrow.apache.org> | https://github.com/apache/arrow-rs | Apache-2.0 |  | Apache Arrow Flight |
+| app | 0.9.0-alpha |  | https://github.com/spiceai/spiceai | Apache-2.0 |  | All loaded components from the root Spicepod and its dependencies |
```
Comment (lukekim) [inline acknowledgements.md]: Anything that starts with our github

File: acknowledgements.md (hunk 1 above)
Comment (ewgenius) [reply acknowledgements.md]: removed all internal spiceai packages

File: acknowledgements.md (hunk 2 above)
Comment (ewgenius) [reply acknowledgements.md]: also removed

**Review by sgrebnov — APPROVED** [review]

**Review by digadeesh — APPROVED** [review]

**Pull request merged**